main.js

```diff
@@ -14,7 +14,7 @@ function createWindow() {
   mainWindow = new BrowserWindow({
     width: 1200,
     height: 760,
-    minWidth: 1024,
+    minWidth: 1170,
```

Comment:
I changed the minimum width to match the smallest size where the chat can be open and still have room for the userPage.